releasetool/data/playbooks/dummy/dummy.yaml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts:
```

Comment:
I assume the dummy stuff will go away before exciting draft status?

Reply:
yes